core/templates/app.constants.ajs.ts

```diff
@@ -49,6 +49,9 @@ angular.module('oppia').constant(
 angular.module('oppia').constant(
   'EXPLORATION_VERSION_DATA_URL_TEMPLATE',
   AppConstants.EXPLORATION_VERSION_DATA_URL_TEMPLATE);
+angular.module('oppia').constant(
+  'ENTITY_TRANSLATIONS_HANDLER_URL_TEMPLATE',
+  AppConstants.ENTITY_TRANSLATIONS_HANDLER_URL_TEMPLATE);
```

Comment:
is this really required? Can't we directly use the value from app constants instead of injecting it?

Reply:
I'm just following the existing pattern and all other constants are added similarly, it can be done directly but I'm not sure why we follow this pattern. Let me know if you want me to follow this new pattern.

Reply:
I would really stress on avoiding any angularjs related change. This is one of those changes in my opinion. Lets not go with anything that adds more to angularjs. (This one is pretty straightforward to avoid so lets avoid)

Reply:
Removed!